core/src/main/java/io/cucumber/core/feature/FeaturePath.java

```diff
@@ -27,6 +32,10 @@
  */
 public class FeaturePath {
 
+    private static final Logger log = LoggerFactory.getLogger(FeaturePath.class);
+
+    private static final Pattern FILESYSTEM_PATH_TO_RESOURCES = Pattern.compile("/*src/(?:main|test)/resources/(.*)");
```

Comment:
This pattern suggests that `/src/test/resources/com/example` is a file system path to a package. But I think it is unlikely that the file system root is part of any project. So I don't think we have to warn users about this. They'll be warned about cucumber not being able to find any feature files.

The pattern also suggests that `src/test/resources` is not a path to a package (it's the root package). Consider adding a test case for this.